core/bin/snapshots_creator/src/main.rs

```diff
@@ -71,15 +87,16 @@ async fn main() -> anyhow::Result<()> {
     let _guard = builder.build();
     tracing::info!("Starting snapshots creator");
 
-    let object_store_config =
-        SnapshotsObjectStoreConfig::from_env().context("SnapshotsObjectStoreConfig::from_env()")?;
-    let blob_store = ObjectStoreFactory::new(object_store_config.0)
+    let object_store_config = general_config
+        .core_object_store
+        .context("ubject store config")?;
```

Comment:
```suggestion
        .context("snapshot creator object storage config")?;
```

Reply:
Addressed in https://github.com/matter-labs/zksync-era/pull/2471/commits/846650722923f639950f3291d5e272f35c4a69b3